Documentation/SpatialAwareness/SpatialAwarenessGettingStarted.md

```diff
@@ -0,0 +1,90 @@
+
+### Enable Spatial Awareness
+
+The spatial awareness system is managed by the MixedRealityToolkit object (or another [service registrar](xref:Microsoft.MixedReality.Toolkit.IMixedRealityServiceRegistrar) component). 
+
+```
+Note:
+The following steps presume use of the MixedRealityToolkit object. Steps required for other service registrars may be different.
+```
+1. Select the MixedRealityToolkit object in the scene hierarchy.
+
+![MRTK Configured Scene Hierarchy](../../External/ReadMeImages/MRTK_ConfiguredHierarchy.png)
+
+2. Navigate the Inspector panel to the Spatial Awareness System section and check Enable
+
+![Enable Spatial Awareness](../../External/ReadMeImages/SpatialAwareness/MRTKConfig_SpatialAwareness.png)
+
+3. Select the Spatial Awareness System implementation
+
+![Select the Spatial Awareness System Implementation](../../External/ReadMeImages/SpatialAwareness/SpatialAwarenessSelectSystemType.png)
+
+```
+Note:
+Users of the default profile (DefaultMixedRealityToolkitConfigurationProfile) will have the spatial awareness system pre-configured to use the MixedRealitySpatialAwarenessSystem from the Mixed Reality Toolkit Spatial Awareness Service package.
+```
+
+### Register observers
+
+Before the spatial awareness system can provide applications with data about the real-world, at least one spatial observer must be registered. Spatial observers are generally platform specific components that may vary in the type(s) of data (ex: meshes) provided.
+
+1. Open or expand the Spatial Awareness System profile
+
+![Spatial Awareness System Profile](../../External/ReadMeImages/SpatialAwareness/SpatialAwarenessProfile.png)
+
+2. Click "Add Spatial Observer"
+3. Select the Spatial Observer implementation
+
+![Select the Spatial Observer Implementation](../../External/ReadMeImages/SpatialAwareness/SpatialAwarenessSelectObserver.png)
+
+```
+Note:
+Users of the default profile (DefaultMixedRealitSpatialAwarenessSystemProfile) will have the spatial awareness system pre-configured to use the WindowsMixedRealitySpatialMeshObserver from the Mixed Reality Toolkit Windows Mixed Reality Provider package.
+```
+
+#### Configure observers
+
+Once the spatial observer(s) have been registered with the system, they can be configured to provide the desired data.
+
+```
+Note:
+When configuring a spatial observer, many implementations will auto-populate the observer's configuration profile with common default values.
+```
+
+1. Open or expand the Spatial Observer profile
+
+![Spatial Mesh Observer Profile](../../External/ReadMeImages/SpatialAwareness/SpatialAwarenessMeshObserverProfile.png)
+
+2. Configure the desired options
+
+The illustration in the previous step shows the configuration profile for a spatial mesh observer. Please see [Spatial Mesh Observer profile](../TODO.md) for more information pertaining to the specific settings available to mesh observers. Other observers may have similar settings.
```

Comment:
Chatted about this offline, but:

https://about.gitlab.com/2016/10/11/wrapping-text/

Reply:
Edit 2 - actually this might suck because our tooling isn't great for markdown right now (i.e. this works best when you have auto-wrapping formatters but is actually a pain if you don't have that).

Let's hold off on this style until I have better ideas on our stuffs.